Makefile

```diff
@@ -65,12 +65,12 @@ proto: ## Rebuild protobuf files
 	go run scripts/protobuf.go install
 	go run scripts/protobuf.go generate
 
-##@ SDK
+##@ Simulator
 
-.PHONY: install-sdk
-install-sdk: ## install storj-sdk
+.PHONY: install-sim
+install-sim: ## install storj-sim
 	@echo "Running ${@}"
-	@go install -race -v storj.io/storj/cmd/storj-sdk storj.io/storj/cmd/bootstrap storj.io/storj/cmd/satellite storj.io/storj/cmd/storagenode storj.io/storj/cmd/uplink storj.io/storj/cmd/gateway
+	@go install -race -v storj.io/storj/cmd/storj-sim storj.io/storj/cmd/bootstrap storj.io/storj/cmd/satellite storj.io/storj/cmd/storagenode storj.io/storj/cmd/uplink storj.io/storj/cmd/gateway
```

Comment:
(oops, commented in the wrong place before.) Did you mean to leave off `certificates` here, compared to what was removed in .travis.yml?

Reply:
Yes, `storj-sim` is not using `certificates`, however it shouldn't have been removed from travis.